ga4gh/frontend.py

```diff
@@ -25,6 +26,7 @@
 
 
 app = flask.Flask(__name__)
+urls = []
```

Comment:
Unless there's some other way of sharing information between different parts of `frontend` that I'm not thinking of... 

I'll try and attach the information to some part of the `app` variable, but that may or may not work.